student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] The superclass's (Person) initialize method accepts age, name, and parent_permission as arguments with default values. If the superclass's initialize method has different parameters, you should adjust the child class's initialize method accordingly to match them when calling super. Please change your super like this 👇🏽 

```ruby
super(age, name, parent_permission)
```